### PR TITLE
add eslint to check for at least 1 negative test

### PR DESCRIPTION
**Pull request by marudor**
simple eslint setup + writte plugin that checks for negative tests.
Curently set to throw if we have no negative tests.
We could increase it to a specific amount of negative tests.

### CODE REVIEW

File: definitions/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "parser": "babel-eslint",
+  "plugins": [
+    "flow-typed"
+  ],
+  "rules": {
+    "flow-typed/negative-tests": [2, 1]
```
Comment (thejameskyle) [inline definitions/.eslintrc]: Where is this eslint rule defined?

Comment (marudor) [inline definitions/.eslintrc]: https://github.com/marudor/eslint-plugin-flow-typed

**Comment by ryyppy** [comment]
This is great, but doesn't the `cli/.flowconfig` also require a `suppress_comment` options for `ExpectError`?

**Comment by marudor** [comment]
Nope,
this PR and rule is just to test the tests of the definitions.
All of them are in definitions/
Has nothing to do with the cli

**Comment by marudor** [comment]
@jeffmo whats you're opinion? would merge if you're okay to get it running asap.
new Rules for stricter checking would be major semver updates. That way I don't accidently break flow-typed builds with changes to linting and tests.

**Comment by jeffmo** [comment]
Seems good, but I'm a little surprised eslint can parse everything right now considering https://github.com/flowtype/flow-typed/issues/161... Any idea why that's actually working?

Also, would it make sense to put the eslint rule directly in this repo so that there's less work for Travis to do at CI time (i.e. download another dependency)? It's a small dependency it seems, but npm is pretty slow...so these all start to add up.

**Comment by marudor** [comment]
I'm not parsing the flow types. Only tests.

Would keep the plugin seperate for now. It really is a small package - extra time is nearly 0 compared to actual testing.

**Comment by jeffmo** [comment]
> I'm not parsing the flow types. Only tests.

Ahhh, that explains it!

**Pull request merged**